ci-scripts/run_suite.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Similar request here, can we rename this file to `durable-driver.sh` or similar to make it clear this is for durable tests?